docker/start

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e
+
+if [[ $PGRST_DB_URI_FILE != "" ]]; then 
+    DB_URI=$(<$PGRST_DB_URI_FILE); 
+else
+    DB_URI=${PGRST_DB_URI};
+fi
+
+if [[ $PGRST_JWT_SECRET_FILE != "" ]]; then 
+    JWT_SECRET=$(<$PGRST_JWT_SECRET_FILE); 
+else
+    JWT_SECRET=${PGRST_JWT_SECRET};
+fi
+
+CONF=/etc/postgrest-secrets.conf
+touch $CONF
+cat > $CONF <<-EOS
```

Comment:
You could replace this 3 lines with:
```bash
cat <<EOF > postgrest.conf
```

Also let's keep the name as `postgrest.conf`, not all the config values are secrets.